subsys/bluetooth/audio/bap_endpoint.h

```diff
@@ -93,6 +98,7 @@ struct bt_bap_broadcast_source {
 #if CONFIG_BT_AUDIO_CODEC_CFG_MAX_DATA_SIZE > 0
 	/* The codec specific configured data for each stream in the subgroup */
 	struct bt_audio_broadcast_stream_data stream_data[BROADCAST_STREAM_CNT];
+	struct bt_bap_broadcast_source_bis bis[BROADCAST_STREAM_CNT];
```

Comment:
`bt_audio_codec_cfg` may still contain metadata even if `CONFIG_BT_AUDIO_CODEC_CFG_MAX_DATA_SIZE` so it's incorrectly guarded